lib/models/plugins/addCRUDFunctions.js

```diff
@@ -171,6 +171,7 @@ async function getConnection({
 
   const direction = (before) ? BACKWARDS : FORWARDS;
 
+  // TODO: Blocks of code in if and elseif statements are equal.
```

Comment:
Looks like the else if block below should have set the cursor to before instead of after, can you please change that @PrinceWaune and remove this TODO?

Reply:
> This is core functionality. I don’t think it’s a good idea to edit this here

I agree, in this case, since it's a small change, let's just do it here to save us some time and management.

Reply:
We don't paginate backwards in our UI anywhere so this isn't something QA will have tested previously.

Reply:
> Looks like the else if block below should have set the cursor to before instead of after, can you please change that @PrinceWaune and remove this TODO?

Yeah, sure!